File: qc-domain/src/main/java/com/eager/questioncloud/domain/post/PostCommentDetail.java
package com.eager.questioncloud.domain.post;

import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Getter
@AllArgsConstructor
@Builder
public class PostCommentDetail {
    private Long id;
    private String writerName;
    private String profileImage;
    private String comment;
    private Boolean isCreator;
    private Boolean isWriter;
    private LocalDateTime createdAt;
}

File: qc-domain/qc-post/src/main/kotlin/com/eager/questioncloud/post/dto/PostCommentDetail.kt
package com.eager.questioncloud.post.dto

import java.time.LocalDateTime

class PostCommentDetail(
    val id: Long?,
    val writerName: String?,
    val profileImage: String?,
    val comment: String?,
    val isCreator: Boolean?,
    val isWriter: Boolean?,
    val createdAt: LocalDateTime?,
)
